Index: src/it/develhope/Cazzeggio/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/it/develhope/Cazzeggio/Main.java b/src/it/develhope/Cazzeggio/Main.java
new file mode 100644
--- /dev/null	(date 1675639601739)
+++ b/src/it/develhope/Cazzeggio/Main.java	(date 1675639601739)
@@ -0,0 +1,9 @@
+package it.develhope.Cazzeggio;
+
+public class Main {
+    public static void main(String[] args) {
+
+
+
+    }
+}
Index: src/it/develhope/Cazzeggio/Cazzeggio.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/it/develhope/Cazzeggio/Cazzeggio.java b/src/it/develhope/Cazzeggio/Cazzeggio.java
new file mode 100644
--- /dev/null	(date 1675639601765)
+++ b/src/it/develhope/Cazzeggio/Cazzeggio.java	(date 1675639601765)
@@ -0,0 +1,5 @@
+package it.develhope.Cazzeggio;
+
+public class Cazzeggio {
+
+}
